spec/javascripts/controllers/containers/container_topology_controller_spec.js

```diff
@@ -27,4 +27,11 @@ describe('containerTopologyController', function() {
             expect($scope.kinds).not.toBe(undefined);
         });
     });
+
+    describe('kinds contain all expected kinds', function() {
+        it('in all main objects', function() {
+               expect(Object.keys($scope.kinds).length).toBeGreaterThan(7);
+               expect($scope.kinds["Container"]).not.toBe(undefined);
```

Comment:
> I'm not sure how jasmine handles .toBe(undefined) - because it is the equivalent of .toBe() unless you look at arguments.length.
> 
> I think you're supposed to use .toBeUndefined().

thanks, checking it.
I was following the convention  in lines 25-27 that does ".not.toBe(undefined)"